### DISCUSSION TITLE
Make Preferred Price Source explicitly searchable

### DISCUSSION THREAD

**Discussion by jacobburrell** [btcpayserver/btcpayserver#4189] [Ideas / Feature Requests]
This dropdown list for [Preferred Price Source](https://docs.btcpayserver.org/FAQ/Stores/#how-to-change-the-exchange-rate-provider-for-invoices) is too long to be useful as a dropdown.

In the future it likely would only grow in size, making searching for a specific exchange useful.
While the current drop down does have hidden search by typing, many users, such as mobile users might not know how to access the search functionality.

Perhaps something like [Select2](https://select2.org/getting-started/basic-usage) can be implemented for this and any large (30+ item) dropdowns to make it easy to select what the user is looking for.

**Comment by dennisreimann** [comment]
Good point. We can make it a [datalist](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/datalist) like we already have with the currency selection.
